README.md

```diff
@@ -70,6 +70,29 @@ The operator covers the following cases:
   * Remove the route URL from the OAuth client authorized redirect URIs in `cluster` deployment
   * Remove the generated client certificate Secret
 
+### Custom TLS route certificate
+TLS certificate for the created route is generated by default by Openshift, however it's possible to provide 
+a own certificate stored in generic secret and point to it in the CR:
+example:
+```
+oc create secret generic route-custom-cert --from-file=cert=tls.crt --from-file=key=tls.key --from-file=caCert=caCert.crt --from-file=destinationCaCert=destinationCaCert.crt
+```
+```yaml
+apiVersion: hawt.io/v1alpha1
+kind: Hawtio
+metadata:
+  name: hawtio-online
+spec:
+...
+  routeTLSSecret: route-custom-cert
+... 
+```
+The operator is looking for the following keys:
+ - `cert` - mandatory
+ - `key` - mandatory
+ - `caCert` - optional 
+ - `destinationCaCert` - optional
```

Comment:
Why would that be used? As the operator relies on the service signing certificate service, the destination CA is injected.

Reply:
yeah so I've technically just copied what can be set when re-encript tls termination is used. It can be always blank:)

Reply:
If it's set, then it's basically broken. It has to be injected from the service signing certificate service CA.